packages/sw-build/src/lib/generate-sw.js

```diff
@@ -43,6 +43,9 @@ const errors = require('./errors');
  * @param {String} [input.cacheId] An optional ID to be prepended to caches
  * used by sw-build. This is primarily useful for local development where
  * multiple sites may be served from `http://localhost`.
+ * @param {Boolean} [input.handleFetch] Stops the generated service worker
```

Comment:
This JSDoc could use some work. How about:

"If `true`, the service worker's fetch handlers will respond to requests. If `false`, the fetch handlers will not respond, which can be useful in a development environment. (Defaults to `true`.)"